go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/coder/terraform-provider-coder/v2
 
-go 1.22.9
+go 1.23.0
```

Comment:
This is burying the lede IMHO.

Upgrading to a new minor should be done in its own PR.

Reply:
This was done automatically by `go mod tidy` and causing CI to fail.

Reply:
@dependabot recreate